.github/workflows/README.md

```diff
@@ -0,0 +1,63 @@
+# Our Workflows
+
+## Format Check
+[![Format Check](https://github.com/nasa/osal/actions/workflows/format-check.yml/badge.svg)](https://github.com/nasa/osal/actions/workflows/format-check.yml)
+
+This action uses clang-format-10 to check for format errors.
```

Comment:
Add link to .clang-format config file

Reply:
This link https://releases.llvm.org/10.0.0/tools/clang/docs/ClangFormat.html?

Reply:
Used https://github.com/nasa/cFS/blob/main/.clang-format